apps/rush-lib/src/logic/PathTree.ts

```diff
@@ -0,0 +1,117 @@
+/**
+ * @public
```

Comment:
This does not seem to be actually public

Reply:
Fixed.